scripts/build.sh

```diff
@@ -296,6 +296,18 @@ case "$product-$platform-$method" in
 
   Auth-*-xcodebuild)
     if check_secrets; then
+      RunXcodebuild \
```

Comment:
Redundant since the integration tests below depend on AuthSample - but fine for completeness.

Reply:
Ack, thanks! I will leave this for completeness.